fixture-monkey/src/main/java/com/navercorp/fixturemonkey/arbitrary/ArbitraryNode.java

```diff
@@ -275,13 +317,23 @@ public ContainerSizeConstraint getContainerSizeConstraint() {
 		return this.status.postArbitraryManipulators;
 	}
 
+	@Deprecated
 	public boolean isFixed() {
-		return getStatus().isFixed();
+		return false;
+	}
+
+	public boolean isFixedAsNull() {
+		return getStatus().isFixedAsNull();
 	}
 
 	@SuppressWarnings("BooleanMethodIsAlwaysInverted")
 	public boolean isLeafNode() {
-		return this.getChildren().isEmpty() && this.getArbitrary() != null;
+		return this.isFixedAsNull()
+			|| ((this.getChildren().isEmpty() || isMap()) && this.getArbitrary() != null);
+	}
+
+	public boolean isMap() {
```

Comment:
여기도 마찬가지로 public 으로 노출되는 신규 메소드 이므로, `@API(since = "0.4.0", status = Status.EXPERIMENTAL)` 가 있으면 좋을거 같습니다.

public 으로 노출되는 신규 메소드는 해당 버전과 EXPERIMENTAL 을 달아놓고, 그 다음 버전으로 올라가면서 MAINTAIN 으로 상태를 변경하면 좋을거 같습니다.